pkg/jmxfetch/jmxfetch.go

```diff
@@ -248,8 +232,7 @@ func (j *JMXFetch) Start(manage bool) error {
 // Stop stops the JMXFetch process
 func (j *JMXFetch) Stop() error {
 	var stopChan chan struct{}
-
-	if j.JmxExitFile == "" {
+	if runtime.GOOS != "windows" {
```

Comment:
just a note for future reference: we typically use platform specific files `foo_windows.go` as opposed to relying on `GOOS` for this sort of thing - where we'd implement the platform-specific `Stop()` on separate files. But, it's not clear to me with the super small windows `Stop()` code this is worth it; it would also mean we'd have to put the non-windows logic in its own file. If you want to do it for consistency, that's fine, but it's probably overkill.

Reply:
Since we already had a `jmxfetch_windows.go` and `jmxfetch_nix.go` files, I've moved it there.